cmd/fleetctl/package.go

```diff
@@ -289,8 +288,9 @@ func packageCommand() *cli.Command {
 				return errors.New("native tooling is only available in Linux")
 			}
 
-			if opt.LocalWixDir != "" && runtime.GOOS != "windows" {
-				return errors.New(`Could not use local WiX to generate an osquery installer. This option is only available on Windows.
+			if opt.LocalWixDir != "" && runtime.GOOS != "windows" && runtime.GOOS != "darwin" {
```

Comment:
do we need a `runtime.GOARCH == arm64` here as well?

Reply:
IIRC we need to be careful with the `runtime.GOARCH == arm64` because it will work during your development (`make fleetctl` and `./build/fleetctl`), but we currently build and release fleetctl as non-universal binary so it runs under Rosetta2, and IIRC `runtime.GOARCH != arm64`.

(My memory may be failing me but worth double checking this.)

Reply:
The flow allows to use Wix on either macOS architecture.

I create a PR for the universal binary, so that we can rely on `runtime.GOARCH`: https://github.com/fleetdm/fleet/pull/16346